--- ngrep-1_47/Makefile.in	2017-09-08 05:28:35.000000000 +0800
+++ ngrep-1_47.mod/Makefile.in	2018-06-26 22:12:38.674617280 +0800
@@ -28,13 +28,13 @@ MANDIR_INSTALL = $(mandir)/man8
 
 INSTALL = ./install-sh
 
-REGEX_OBJS=@REGEX_OBJS@
-REGEX_DIR=@REGEX_DIR@
+REGEX_OBJS=
+REGEX_DIR=
 
 
 all: $(TARGET)
 
-$(TARGET): $(REGEX_OBJS) $(OBJS)
+$(TARGET): $(OBJS)
 	$(CC) $(STRIPFLAG) -o $(TARGET) $(OBJS) $(REGEX_OBJS) $(LDFLAGS) $(LIBS)
 
 debug: $(REGEX_OBJS) $(OBJS)
